src/main/java/sirius/biz/storage/layer2/mongo/MongoBlob.java

```diff
@@ -190,6 +190,12 @@
     public static final Mapping DELETED = Mapping.named("deleted");
     private boolean deleted;
 
+    /**
```

Comment:
The same changes also have to be applied to the SQL parts

Reply:
It's in the pipeline. Was waiting for your review first to avoid double-fixing!